### DISCUSSION TITLE
In Line Edit Color / Row Hover Highlight Settings

### DISCUSSION THREAD

**Discussion by jbk68** [cortexrd/Knack-Toolkit-Library#162] [General]
It appears that the 'in line Edit Color" and row hover Highlight" settings are reversed as of the writing of this post.

If setting is "false" on in line edit color, (2) small icons appear next to search table's search field.  Change to "true" icons disappear.  ???

If setting is "false" on row hover highlight, rows are highlighted.  Change to "true" highlights disappear.  ???

**Comment by cortexrd** [comment]
Hi Jon,

I've tested the two flags and they work as expected.

```
inlineEditColor: false,
rowHoverHighlight: false,
```

1. What are the two icons you're referring to?  Next to **Search** or next to **Add filters**?  If you're talking about the yellow diskette and the pink X, they have nothing to do with the flags above.  They will show up when your grid has Inline editing enabled.

2. About the row how hover in grids, check your settings here:

![image](https://github.com/cortexrd/Knack-Toolkit-Library/assets/42851359/c589778a-9f46-4466-999b-5964f6b83215)

Uncheck the "**Hovering over a row will highlight it**" box AND set the **rowHoverHighlight** flag to **false** in KTL.

Note that Knack always highlight the cell when you hover over it, if the grid has **Inline editing** enabled.

Norm

**Comment by cortexrd** [comment]
Can you provide a screen grab of what is wrong, with the flags values please?

**Comment by jbk68** [comment]
Hey Norm, shortly after initial post on this matter, it's started working properly.  Not sure why the highlighted rows appeared when settings were "false" and disappeared when set to "true".  All the rows were highlighted not just the one you hovered over.  Like I said, it corrected itself shortly after my post and I apologize for not posting it was corrected as I had left my office and was on the road.

As for the icons that appeared, one was an icon of a old school floppy disc and the other was icon with an "X" in it.  Initially they only would disappear when I set one of the settings to "true".  Again, it corrected itself a short period later.

If/when this happens again, I be sure to post screen prints to proof I not complete going insane... LOL!!!

**Comment by cortexrd** [comment]
Ha ha!!!  Both of these happen sometimes:  Knack has intermittent bug _and_ we can go insane!!!

No trouble, let's just keep an eye on this :)

Cheers,
Norm
